src/init.c

```diff
@@ -1264,10 +1268,12 @@ static int hyper_loop(void)
 	return 0;
 }
 
-int main(int argc, char *argv[])
+static int hyper_setup_init_process(void)
 {
-	char *cmdline, *ctl_serial, *tty_serial;
+	if (!is_init)
+		return 0;
```

Comment:
cloud you move the test up (outside this hyper_setup_init_process())?

and it seams better to remove this global is_init since there is no other usage.